src/Player.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Created by Shreyansh97 on 7/9/19
+ */
+
+public class Player {
+  private String name;
+  private int position;
+  private boolean won;
+  
+  public Player(String name) {
+    this.name = name;
+    position = 0;
+    won = false;
+  }
+  
+  public boolean hasWon() {
```

Comment:
Player should not have this info. Game should ask for player's position and decide if player has won.